otherlibs/stdune/dune_filesystem_stubs/dune_filesystem_stubs.ml

```diff
@@ -293,9 +293,9 @@ module Unix_error = struct
     let equal (a1, b1, c1) (a2, b2, c2) =
       equal a1 a2 && String.equal b1 b2 && String.equal c1 c2
 
-    let to_string (error, syscall, arg) =
-      Format.sprintf "Unix error (%s, %s, %s)" (Unix.error_message error)
-        syscall arg
+    let to_string_hum ?(prefix = "") (error, syscall, arg) =
```

Comment:
`prefix` feels like stuffing a feature that's not necessary: we could either do `prefix ^ to_string_hum ...` or `Pp.text prefix ++ Pp.error` at the various call sites and it would be just as clear

Reply:
For `to_string_hum`, I agree that `prefix` isn't really useful, so I'll drop it.

For `Pp.error` it's a bit trickier, since I want the result to be inside a single `Pp.verbatim` to avoid unnecessary line breaks.

> `Pp.text prefix ++ Pp.error` at the various call sites

You probably meant `Pp.concat ~sep:Pp.space [Pp.text prefix; Pp.error]` which is a bit longer but is also different because it may introduce a line break. It's not a big deal but still it feels nicer to be able to just say `Pp.error ~prefix ...` instead.

Reply:
Ah, I missed the `Pp.verbatim`. You can use `Pp.hbox` to prevent line breaks. Either way, we started using `Pp` everywhere to get better formatting and avoid super long lines, so it feels like defeating the purpose...

Reply:
But for debug message, good formatting is less important and printing on a single makes grepping easier. So I can why you would want that. Maybe we should instead add a different printing functions for such messages?

Reply:
Also, this PR is already an improvement and what we are discussing here is less important, so we can leave that for later. We should still move the  pretty printer for detailed errors to its own module though.

Reply:
OK, I dropped `prefix` for `to_string_hum` but kept it in `Unix_error.Detailed.pp`. This seems like a good state for now.